google-cloud-clients/google-cloud-compute/src/test/java/com/google/cloud/compute/v1/it/ITComputeTest.java

```diff
@@ -85,6 +101,26 @@ public void testGetDiskType() {
     assertThat(diskType.getDefaultDiskSizeGb()).isNotNull();
   }
 
+  @Test
+  public void testInsertInstance() {
```

Comment:
Do we have a test that asserts a successfully inserted instance?

Reply:
It'll be in https://github.com/googleapis/google-cloud-java/pull/4420/files#diff-72d767a100f949c4606b04bb6820a8feR248. That is blocked on an LRO implementation tho.

Reply:
So this should be fine for now, and a later PR will test inserting an instance.